src/Angor/Client/Pages/Settings.razor

```diff
@@ -96,7 +97,8 @@
                 {
                     <tr>
                         <td><a href="@relay.Url" target="_blank">@relay.Url</a></td>
-                            <td style="color: @(relay.Status == UrlStatus.Online ? "green" : relay.Status == UrlStatus.NotReady ? "yellow" :  "red");">@relay.Status.ToString()</td>
+                        <td>@relay.Name</td>
```

Comment:
A relay has a name? is it assigned or fetched form the relay?

Reply:
Yes, I have updated the code to do that